src/languages/php.js

```diff
@@ -12,13 +12,15 @@ Category: common
  * */
 export default function(hljs) {
   const regex = hljs.regex;
-  const IDENT_RE_CORE = '[a-zA-Z0-9_\x7f-\xff]*' +
+  const IDENT_RE = '[a-zA-Z_\\x7f-\\xff][a-zA-Z0-9_\x7f-\xff]*' +
```

Comment:
Is it a single `\` or `\\` for `\x7f`?  Looks like we're doing it different in different spots.  Is that intentional?

- L15/20 etc. Lines 15 has both forms.

Reply:
Good catch. Smart paste tried to be helpful. Fixed

Reply:
I made this typo in earlier PR :(